driver/src/bin/main.rs

```diff
@@ -1,15 +1,13 @@
-extern crate env_logger;
 extern crate graph;
 extern crate simple_logger;
 
 use dfusion_core::database::GraphReader;
 
 use driver::contract::SnappContractImpl;
-use driver::price_finding::{
-    NaiveSolver,
-    LinearOptimisationPriceFinder,
-    PriceFinding
-};
+use driver::order_driver::OrderProcessor;
+use driver::price_finding::NaiveSolver;
```

Comment:
@fleupold  is there any particular reason we are using 

```
use driver::order_driver::OrderProcessor;
use driver::price_finding::NaiveSolver;
use driver::price_finding::LinearOptimisationPriceFinder;
use driver::price_finding::PriceFinding;
```

over 
```
use driver::price_finding::{
    NaiveSolver,
    LinearOptimisationPriceFinder,
    PriceFinding
};
```

Reply:
No, and I actually prefer the latter. Must have been an oversight when resolving merge conflicts.

Reply:
Cool, I'll come back to this someday and deal with it at some point!